addons/hr_attendance/static/src/components/kiosk_barcode/kiosk_barcode.js

```diff
@@ -20,6 +21,13 @@ export class KioskBarcodeScanner extends BarcodeScanner {
         }
         return super.facingMode;
     }
+
+    get installURL() {
+        const url = `hr_attendance/${this.props.token}`
+        return `/scoped_app?app_id=hr_attendance&path=${encodeURIComponent(
+            url.replace(document.location.origin + "/", "")
+        )}`;
```

Comment:
Not sure why are we using replace over here 
can't you just use the URL directly 
can you give it a try an let me know :blush:

Reply:
I was rewriting the function from the base, but we can use the URL directly here since it is already clean.